gstools/variogram/variogram.py

```diff
@@ -20,12 +22,29 @@
     generate_grid,
 )
 from gstools.variogram.binning import standard_bins
-from gstools.variogram.estimator import (
-    directional,
-    ma_structured,
-    structured,
-    unstructured,
-)
+
+if config.USE_RUST:
+    # pylint: disable=E0401
+    from gstools_core import (
```

Comment:
Shouldn't it be possible [to merge those](https://stackoverflow.com/questions/29865421/how-can-i-import-multiple-items-from-a-module-and-rename-them-in-python) or this is a formatter issue?

Reply:
Yeah... that was `isort`'s idea. It's about as ugly as it gets, right?! :-D And `black` and `isort` can't even agree on where to put the `# pragma: no cover`.

Reply:
What about setting `# pragma: no cover` behind `if config.USE_RUST:` once?
Also the `# pylint: disable=C0412` is redundant (see line 13) and we could also just add `E0401` in line 13.

Reply:
Sorry, I think I was a bit annoyed yesterday by the combination of a false positive coming from pylint and then the isort and black peskiness. I imnproved the situation, but it seems that isort does not like grouped renaming of imports.
I don't want to disable the pylint error `E0401` globally, as import errors can be quite a severe problem.